aspnetcore/toc.yml

```diff
@@ -325,8 +325,20 @@
       uid: signalr/version-differences
     - name: WebSockets without SignalR
       uid: fundamentals/websockets
+- name: gRPC
```

Comment:
I added this doc to its own top level section. Maybe there's a better place to put it?

Reply:
@tdykstra

Reply:
The Advanced section is probably the best location for it.

Reply:
Some context: gRPC is a new framework for ASP.NET Core that will sit alongside MVC and SignalR as options. This PR is the beginning, and it will have many sub-topics and tutorials added in the future.

I think it should be at the top level, but perhaps renamed to something like "Remote Procedure Call apps" or "RPC apps"?

Reply:
The introduction should start out by explaining what it is and how it fits into the scheme of things in ASP.NET Core; at present it assumes you already know what the technology/scenario is all about and jumps into technical details.  Does such an explanation exist somewhere that we could pull into the intro doc?

Regarding the node name, we try to avoid acronyms, so Remote Procedure Call apps would probably work and I'd locate it after SignalR (Real-time apps) in the ToC.

Reply:
Ok!

> Does such an explanation exist somewhere that we could pull into the intro doc?

Not at the moment but we'll write one. Is the [What is SignalR?](https://docs.microsoft.com/en-us/aspnet/core/signalr/introduction?view=aspnetcore-2.2#what-is-signalr) a good guide to follow for its content?

Reply:
I'll rename the node name. Any examples of what you want to cover in the introduction? I presumed a link to the grpc docs page would be sufficient for explaining what it is and the scenarios it's used for but maybe that needs elaboration? There's not too much ASP.NET Core specific since we are implementing the framework based on their specs. Notable difference are covered later on in the docs.

Reply:
Yes. that's a good example.

Reply:
Added a new intro.